resources/build-npm.js

```diff
@@ -107,6 +125,8 @@ function buildPackageJSON() {
   delete packageJSON.scripts;
   delete packageJSON.devDependencies;
 
+  packageJSON.type = 'commonjs';
```

Comment:
This is probably a good idea to make it clear that all bundled files with a `.js` extension are CommonJS.

Reply:
agreed!